CHANGELOG.md

```diff
@@ -1,5 +1,34 @@
 # PD Change Log
 
+## v3.0.0
+
++ Support re-creating a cluster from a single node 
++ Migrate Region metadata from etcd to the go-leveldb storage engine to solve the storage bottleneck in etcd for large-scale clusters 
++ API 
+  - Add the `remove-tombstone` API to clear Tombstone stores
+  - Add the `ScanRegions` API to batch query Region information
+  - Add the `GetOperator` API to query running operators
+  - Optimize the performance of the `GetStores` API
++ Configurations
+  - Optimize configuration check logic to avoid configuration item errors
+  - Add `enable-two-way-merge` to control the direction of Region merge 
```

Comment:
It should be `enable-one-way-merge` here

Reply:
I just copy from release note. I guess release note needs to be changed as well.